### PR TITLE
Switching to Community-Spec-1.0 for specification license

### PR DESCRIPTION
**Pull request by idunbarh**
See issue !13.

LF started IP/License review of SBOMit for the OpenSSF sandbox application, and [requested we switch the license of the spec to Community Specification License 1.0](https://github.com/ossf/tac/issues/191#issuecomment-1692546537)

See https://github.com/CommunitySpecification/Community_Specification/blob/main/1._Community_Specification_License-v1.md
This is required by the OpenSSF Charter (Section 5, Page 9):
https://cdn.platform.linuxfoundation.org/agreements/openssf.pdf

### CODE REVIEW

**Review by trishankatdatadog — APPROVED** [review]

**Comment by sudo-bmitch** [comment]
Does this require getting signoff from all committers to the project (to change the license they used with their contributions)?

**Comment by colek42** [comment]
lgtm

**Review by JustinCappos — CHANGES_REQUESTED** [review]
For now, do not merge this PR!

There is still a discussion ongoing and some of our early folks were very opposed to this license.  I'm following up with the OpenSSF in the linked issue on their tracker.

**Review by trishankatdatadog — APPROVED** [review]

**Comment by JustinCappos** [comment]
Had a great conversation with Mike Dolan and Jory about the details here.  We can merge!

**Review by JustinCappos — APPROVED** [review]

**Pull request merged**
